openhtf/__init__.py

```diff
@@ -86,7 +86,7 @@ def __call__(self, test_record):  # pylint: disable=invalid-name
 
 class TestPhaseOptions(mutablerecords.Record(
     'TestPhaseOptions', [], {'timeout_s': None, 'run_if': None})):
-  """
+  """Test Phase Options object.
```

Comment:
this is not a useful docstring, all you did was add spaces between the words already in the name and add 'object' after it -.-